### ISSUE TITLE
Retrieve velocity graph from VelocityKernel

### ISSUE THREAD

**Issue by dawe** [theislab/cellrank#805]
Hello all, as far as I understand the `VelocityKernel` calculates correlations between the velocity vector of any cell with the expression level of neighbor cells, then this is used to infer a transition probability, being the final transition matrix. Is it possible to retrieve the velocity graph that is calculated in the middle or to recalculate it from the transition matrix? Of course, I can recalculate it with `scv.tl.velocity_graph`, but I'm lazy and I would like to spare computations :-)

**Comment by michalk8** [comment]
Hi @dawe ,

`VelocityKernel` has attribute `.logits`, which are the non-normalized (right before softmax) cosine/correlation/dot product similarities between the gene-expression displacement and the velocities. However, I've phased this out in #791, since it wasn't being used anywhere.
If you're using this somewhere downstream, can keep it there for ease of access.

**Comment by Marius1311** [comment]
Hi @dawe, your input would be appreciated. Thanks!

**Comment by dawe** [comment]
Hi @Marius1311 , sorry, I haven’t seen the answer by @michalk8. I would definitely keep that or keep track of parameters useful to invert the transform. If cosines/correlations are transformed to probabilities using softmax, would logarithm of transition matrix do the job?
